packages/ecdsa-secp256k1-verification-key-2019/src/keyUtils.ts

```diff
@@ -0,0 +1,129 @@
+  privateKeyHex: (privateKeyHex: string): string =>
+    Buffer.from(secp256k1.publicKeyCreate(new Uint8Array(Buffer.from(privateKeyHex, 'hex')))).toString('hex'),
+}
+
+export const privateKeyHexFrom = {
+  privateKeyBase58: (privateKeyBase58: string): string => Buffer.from(base58.decode(privateKeyBase58)).toString('hex'),
+  privateKeyJWK: (jwk: PrivateKeyJWK): string =>
+    keyto
+      .from(
+        {
+          ...jwk,
+          crv: 'K-256',
+        },
+        'jwk',
+      )
+      .toString('blk', 'private'),
+  privateKeyUint8Array: (privateKeyUint8Array: Uint8Array): string => Buffer.from(privateKeyUint8Array).toString('hex'),
+}
+
+export const publicKeyUint8ArrayFrom = {
+  publicKeyBase58: (publicKeyBase58: string): Uint8Array => base58.decode(publicKeyBase58),
+  publicKeyHex: (publicKeyHex: string): Uint8Array => Uint8Array.from(Buffer.from(publicKeyHex, 'hex')),
+  publicKeyJWK: (jwk: PublicKeyJWK): Uint8Array => {
+    let asBuffer = Buffer.from(publicKeyHexFrom.publicKeyJWK(jwk), 'hex')
+    let padding = 32 - asBuffer.length
+    while (padding > 0) {
+      asBuffer = Buffer.concat([Buffer.from('00', 'hex'), asBuffer])
+      padding -= 1
+    }
+    return Uint8Array.from(asBuffer)
+  },
+  privateKeyUint8Array: (privateKeyUint8Array: Uint8Array): Uint8Array => secp256k1.publicKeyCreate(privateKeyUint8Array),
+}
+
+export const privateKeyUint8ArrayFrom = {
+  privateKeyBase58: (privateKeyBase58: string): Uint8Array => base58.decode(privateKeyBase58),
+  privateKeyHex: (privateKeyHex: string): Uint8Array => Uint8Array.from(Buffer.from(privateKeyHex, 'hex')),
+  privateKeyJWK: (jwk: PrivateKeyJWK): Uint8Array => {
+    let asBuffer = Buffer.from(privateKeyHexFrom.privateKeyJWK(jwk), 'hex')
+    let padding = 32 - asBuffer.length
+    while (padding > 0) {
+      asBuffer = Buffer.concat([Buffer.from('00', 'hex'), asBuffer])
+      padding -= 1
+    }
+    return Uint8Array.from(asBuffer)
+  },
+}
+
+export const publicKeyJWKFrom = {
+  publicKeyBase58: (publicKeybase58: string, kid: string): PublicKeyJWK =>
+    publicKeyJWKFrom.publicKeyHex(Buffer.from(base58.decode(publicKeybase58)).toString('hex'), kid),
+  publicKeyHex: (publicKeyHex: string, kid: string): PublicKeyJWK => {
+    const key =
+      publicKeyHex.length === compressedHexEncodedPublicKeyLength
+        ? Buffer.from(secp256k1.publicKeyConvert(Buffer.from(publicKeyHex, 'hex'), false)).toString('hex')
+        : publicKeyHex
+
+    return {
+      ...keyto.from(key, 'blk').toJwk('public'),
+      crv: 'secp256k1',
+      kid,
+    }
+  },
+  publicKeyUint8Array: (publicKeyUint8Array: Uint8Array, kid: string): PublicKeyJWK =>
+    publicKeyJWKFrom.publicKeyHex(Buffer.from(publicKeyUint8Array).toString('hex'), kid),
+  privateKeyJWK: (privateKeyJWK: PrivateKeyJWK): PublicKeyJWK => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { d, ...publicKeyJWK } = privateKeyJWK
```

Comment:
Can you get around this by doing `d: _d` or does that still end up as something eslint catches?

Reply:
No for some reason that doesn't work :(